public/app/features/alerting/components/UnifiedAlertingPromotion.test.tsx

```diff
@@ -11,7 +12,8 @@ describe('Unified Alerting promotion', () => {
 
   it('should show by default', () => {
     const promotion = render(<UnifiedAlertingPromotion />);
-    expect(promotion.queryByLabelText('Alert info')).toBeInTheDocument();
+    console.log(selectors.components.Alert.alertV2('info'));
+    expect(promotion.queryByTestId(selectors.components.Alert.alertV2('info'))).toBeInTheDocument();
```

Comment:
The pattern for react testing library is to test as the user users the application, rather than testing implementation details like `data-testid` (and ignoring how not-great the previous label assertion was!) https://testing-library.com/docs/guiding-principles

Is there user-visible text to assert on instead?

Reply:
Yep, I can definitely update the assertion to check if the text content is in the document instead 👍